app/services/core/persistent-stateful-service.ts

```diff
@@ -20,7 +20,7 @@ export abstract class PersistentStatefulService<TState extends object> extends S
   static defaultState = {};
 
   static get initialState() {
-    const persisted = JSON.parse(localStorage.getItem(this.localStorageKey)) || {};
+    const persisted = JSON.parse(localStorage.getItem(this.localStorageKey) as string) || {};
```

Comment:
Is type casting really the right way to handle this?  `localStorage.getItem` really can return `null`.

Reply:
The right way here is to do nothing.
`JSON.parse` works with `null` without exceptions. However, most of native .ts` files don't assume that we use the `strictNullsCheck` mode

Reply:
Ah yes I see.  `JSON.parse(null)` just returns `null`.